pkg/cmd/errors/login.go

```diff
@@ -0,0 +1,49 @@
+package errors
```

Comment:
@fabianofranz 
> I'm not confident this one belongs here too, once this is specific to login which is under oc in pkg/cmd/cli/cmd. Could probably stay in pkg/cmd/cli/cmd/errors

I think that makes sense, @liggitt or @deads2k do you have an opinion on this?